etc/module.xml

```diff
@@ -10,7 +10,7 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Acquired_Payments" setup_version="0.9.8">
+    <module name="Acquired_Payments" setup_version="0.9.7">
```

Comment:
This version has gone backwards

Reply:
@adamcox-acquired fixed in latest commit